src/resources/2006/CommaChameleon.ts

```diff
@@ -0,0 +1,31 @@
+import { $familiar } from "../../template-string.js";
+import { have as _have } from "../../lib.js";
+import { Familiar, familiarEquipment, toInt, visitUrl } from "kolmafia";
+
+const familiar = $familiar`Comma Chameleon`;
+
+/**
+ * Determines whether the player has the Comma Chameleon in their
+ * terrarium
+ *
+ * @returns Whether the player has a Comma Chameleon
+ */
+export function have(): boolean {
+  return _have(familiar);
+}
+
+/**
+ * Transforms Comma Chameleon into the familiar of choice
+ * @param fam determines what to transform into
+ * @returns Whether Comma has been successfully transformed
+ */
+export function transform(fam: Familiar): boolean {
+  let equipment = null;
+  equipment = familiarEquipment(fam);
+  if (!equipment) return false;
+  if (!_have(equipment)) return false;
```

Comment:
```suggestion
  const equipment = familiarEquipment(fam);
  if (equipment !== $item.none) return false;
  if (!have_(equipment)) return false;
```

In general it's good to try to avoid using `let`; here, there's literally no utility to it given that you assign the variable and then _immediately_ reassign it.

`$item.none` isn't falsy, and `familiarEquipment` returns it in the null case (this will be true for virtually all ash functions)